docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+k8ssandra-kube-prometheus-operator-5bdd97c666-qz5vv   1/1     Running     0          6m36s
+k8ssandra-medusa-operator-d766d5b66-wjt7j             1/1     Running     0          6m36s
+k8ssandra-reaper-5f9bbfc989-j59xk                     1/1     Running     0          2m48s
+k8ssandra-reaper-operator-858cd89bdd-7gfjj            1/1     Running     0          6m36s
+k8ssandra-reaper-schema-4gshj                         0/1     Completed   0          3m3s
+prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running     1          6m32s
+```
+
+# Create some data and back it up
+
+Next, create some data in Cassandra by creating a **test_data.cql** file:
+
+```
+CREATE KEYSPACE medusa_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+USE medusa_test;
+CREATE TABLE users (email text primary key, name text, state text);
+insert into users (email, name, state) values ('alice@example.com', 'Alice Smith', 'TX');
+insert into users (email, name, state) values ('bob@example.com', 'Bob Jones', 'VA');
+insert into users (email, name, state) values ('carol@example.com', 'Carol Jackson', 'CA');
+insert into users (email, name, state) values ('david@example.com', 'David Yang', 'NV');
+```
+
+And copy it into the Cassandra pod (the StatefulSet one, which contains <code>-<strong>sts</strong>-</code> in its name):
+
+```
+kubectl cp test_data.cql k8ssandra-dc1-default-sts-0:/tmp -n k8ssandra -c cassandra
+```
+
+Now extract the password to access Cassandra with the k8ssandra-superuser (the password is different for each installation unless it is explicitly set at install time): 
+
+```
+% kubectl get secret k8ssandra-superuser -n k8ssandra -o jsonpath="{.data.password}" | base64 --decode ; echo
+
+XHsZ943WBg5RPNhVAT8x
+```
+
+Let’s now run the uploaded cql script and check that you can read the data (don’t forget to replace the password with the one extracted in the previous step):
+
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -f /tmp/test_data.cql
+
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "SELECT * FROM medusa_test.users"
+
+ email             | name          | state
+-------------------+---------------+-------
+ alice@example.com |   Alice Smith |    TX
+   bob@example.com |     Bob Jones |    VA
+ david@example.com |    David Yang |    NV
+ carol@example.com | Carol Jackson |    CA
+
+(4 rows)
+```
+
+Now backup this data, and check that files get created in your MinIO bucket. 
+
+To that end, use the following command:
+
+```
+helm install my-backup k8ssandra/backup -n k8ssandra --set name=backup1,cassandraDatacenter.name=dc1
+```
+
+Since the backup operation is asynchronous, you can monitor its completion by running the following command:
+
+```
+kubectl get cassandrabackup backup1 -n k8ssandra -o jsonpath={.status.finishTime}
```

Comment:
Do we have any sort of condition we can wait on? (Looking back at the ready one liner for cassdc)

Reply:
Not that I know of.  @jsanda maybe?

Reply:
The CassandraBackup CRD does not yet expose any conditions.

Reply:
I'll pop something on the roadmap, that would be a neat feature.